components/colorloupe/stories/colorloupe.stories.js

```diff
@@ -6,6 +6,12 @@ import { ColorLoupeGroup } from "./colorloupe.test.js";
 
 /**
  * The color loupe component shows the output color that would otherwise be covered by a cursor, stylus, or finger during color selection.
+ *
+ * ## Usage notes
+ *
+ * - Set the `--spectrum-picked-color` custom property to the CSS color value you want to show.
+ * - Implementations must apply the `.is-open` class to display the loupe.
+ * - Color loupe does not have a disabled style. Do not show it when the handle it's attached to is disabled.
```

Comment:
I was going to remove isDisabled from the controls because I think (at least) our template needs to be fixed. According to the documentation, color loupe shouldn't display when its handle is disabled. I went to color handle, turned the `withColorLoupe` to true, then tried to disable it and the color loupe still rendered. 

Think this is a bug? Want me to make a card?

Reply:
Yes great point!  Thanks for catching this and highlighting such a quick fix for it. We can tackle that in the new year for sure.